cmd/dev.go

```diff
@@ -106,6 +106,10 @@ Examples:
 		// Watch for changes
 		watcher, err := dev.NewWatcher(log, dir, theproject.Project.Development.Watch.Files, func(path string) {
 			build()
+			isDeliberateRestart = true
+			log.Debug("killing project server")
+			projectServerCmd.Process.Signal(syscall.SIGTERM)
+			projectServerCmd.Process.Kill()
```

Comment:
this doesn't wait to let the process shutdown cleanly though... this sends the signal and then immediately kills the process. typically you'd want to do something like this:


```golang
projectServerCmd.Process.Signal(syscall.SIGTERM)
ch := make(chan struct{})
go func() {
	projectServerCmd.Wait()
	ch <- struct{}{}
}()
select {
	case <-ch:
		break
	case <-time.After(time.Second*10):
		projectServerCmd.Process.Kill()
}
```